### PR TITLE
Outbrain update

### PR DESCRIPTION
**Pull request by uplne**
1. Outbrain sometimes fails to load. It's probably happening because fastdom will append containers after require returns so this will wrap require call inside fastdom.write
2. unit tests

@jbreckmckye @Calanthe

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js [hunk 1]
```diff
@@ -60,41 +62,35 @@ define([
...
+                || _.contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
+        },
 
-    function init() {
-        mediator.on('modules:commercial:dfp:rendered', function (event) {
-            if (event.slot.getSlotId().getDomId() === 'dfp-ad--merchandising-high'
-                && event.isEmpty && config.page.section !== 'childrens-books-site') {
-                load();
+        init: function () {
+            if (config.switches.outbrain && !config.page.isPreview && !identity.isUserLoggedIn() && config.page.section !== 'childrens-books-site') {
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js]: maybe also check the config.page.shouldHideAdverts flag?

Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js]: That is taken care of in [back end](https://github.com/guardian/frontend/blob/099006dc1a5badeff939a876f2261217551d720a/common/app/views/fragments/contentFooter.scala.html)

File: static/src/javascripts/test/spec/common/commercial/third-party-tags/outbrain.spec.js [hunk 2]
```diff
@@ -1,8 +1,283 @@
...
+                return 'desktop';
+            };
+            sut.getSection = function () {
+                return 'sections';
+            };
+
+            sut.load();
+
+            fastdom.defer(function () {
+                expect($('.OUTBRAIN', $fixtureContainer).first().data('widgetId')).toEqual('AR_12');
```
Comment (jbreckmckye) [inline static/src/javascripts/test/spec/common/commercial/third-party-tags/outbrain.spec.js]: If an expectation fails within a fastdom callback, does it get reported to the test runner?

**Comment by Calanthe** [comment]
:+1:

**Pull request merged**